sql/src/main/java/io/crate/analyze/relations/RelationNormalizer.java

```diff
@@ -419,28 +413,33 @@ public Void visitTwoRelationsUnion(TwoRelationsUnion twoRelationsUnion, UnionPus
         }
 
         private void pushDownOrderBy(QuerySpec querySpec, Optional<OrderBy> orderBy) {
-            querySpec.orderBy(OrderBy.merge(querySpec.orderBy(), orderBy));
+            querySpec.orderBy(OrderBy.merge(querySpec.orderBy(),
+                // The pushedDown OrderBy is passed as InputColumn and must
```

Comment:
at least this comment is not correct anymore since it is now used in general for all types of relations or? at best would be to just use it when it is needed (only Union?)